### DISCUSSION TITLE
ChromeOS Compatible

### DISCUSSION THREAD

**Discussion by Alan-Python** [sebanc/brunch#1256] [Q&A]
Is my Device compatible for my Intel Core i5-7200U CPU 2.50GHz 2712 Mhz 2 Cores 4 Logical Processors

**Comment by 5525Arobotics** [comment]
Should be, but I don't know. Try Brunch r91 and Rammus r91,

**Comment by tejasraman** [comment]
7200U is compatible with Brunch. You can get the latest R94 release and it should work with 7200U. 7200U is a Kabylake(7th gen) so Rammus would be the recommended recovery for you.

**Reply by tejasraman** [reply]
Or nami

**Reply by ghost** [reply]
just wondering which one would work best for an Intel(R) Core(TM) i7-3930K CPU @ 3.20GHz   3.60 GHz  2 gen I think of a desktop.
also I own an ASUS GL 702V laptop, not sure I was not able to get chrome flex to boot on it with USB drive, I going to see if it support it nvidia GTX based 3D card but it looks like it don't. intel i7 core 7th gen , word strix and replublic of gamers on it, no drives, so google chrome should in theory work great if there offical chrome flux supported it. but it doesn't seem to boot that version , I have not tried brunch version on it yet. just the offical one. and no the offical flux doesn't seem to boot on my sandy brige desktop either that one has both an GTX and RTX nvidia card in it, only old dell and toshiba been able to boot the offical flux version.

**Reply by tejasraman** [reply]
Hi @maximalEbooks! An IvyBridge 3930K would work flawlessly with the rammus image (unless it has NVidia graphics of course)

Plugging the cable into the intergrated GPU will make Chrome OS boot.

(Seems like you like NVidia ;) )

**Reply by ghost** [reply]
I have a few devices with NVidia hardware, like that one runs games like epic games store and others of course from windows 11, currently I used redadon 3D cards in the past, one failed on me, so yes I do prefer nvidia just because it seems last a lot longer, its fine for normal linux but of course stuff like chrome os, its a problem, and android x86 runs, but that Bliss OS doesn't like my 3D cards, I do prefer to dual boot stuff some time to time, since I do have 3 hard drives in my computer, stuff like chrome flux I play with but not exactly use in the long time, since its even being hard to get linux to run under it, the installer for linux keeps failing at error starting virtual machine was able to use crosh and get linux on the dell with the intel HD 3D card to run it once, but then when I rebooted it linux was not working again, so I did something wrong, I used ubuntu before without chrome flex os but again it seems very limited on what hardware you can install it on, I been thinking of downgrading my desktop from windows 11 back to windows 10, the startup on that old pc works with windows 11 but it can be a bit slow to load tray icons, stuff like that otherwise the only other problem I had with it was DPC watchdog violation problem had happened on youtube when bringing my desktop out of hibernate mode under the microsoft edge chrome browser which has just started happening within the last month, I think a windows 11 update is causing the problem, since all things like sfc /scannow say all if fine with my desktop. under linux I been able to run some epic games store games, and a few others but you have to run them with wine, which can work but not for directx 12 features I don't think.

Well thanks for reply back to me :)

**Reply by tejasraman** [reply]
Ask me... I've had some very long lasting Radeons in the past, but same with NVidia. Have a 6800 and it still works (although I don't use it)

**Reply by tejasraman** [reply]
Oh, and for wine try DXVK ;)

**Reply by ghost** [reply]
that's cool, I like running myst online uru online game with my old Radeon card, I got ride of it when it had failed. but its been years since I owned one.

**Reply by tejasraman** [reply]
So most Linuxes have a driver called _nouveau_. Nouveau is an open source NVidia driver implementation. 

Bottom line - Chrome OS doesn't have Nouveau or NVidia's Linux driver.

If you want to run Chrome OS, either backport drivers (please don't try - too much weirdness) or don't use the NVidia card.

**Reply by ghost** [reply]
the desktop only has nvidia cards, it doesn't have an intergraded 3D card, it's a gigabyte motherboard, Intel(R) Core(TM) i7-3930K CPU @ 3.20GHz 3.60 GHz 2 gen, me and my dad help me put it together years ago. last bios for it was in 2013.  I would have to have a version of it that with custom packed with an nvidia driver for chrome flux to even boot on its USB disc, otherwise yes this desktop would support chrome OS I think. it has all the bios settings needed for emulators in this motherboard, I just don't own anything else 3D card wise that I could put in it, thats not nvidia based.

https://www.gigabyte.com/Motherboard/G1Assassin-2-rev-10/support#support-dl-driver

**Comment by ghost** [comment]
wine DXVK is that a directx 12 mode, I heard they were working on addon it, but I am not currently playing with linux besides trying to get it running on that dell laptop with it, I also used arch linux and debian before but I do prefer ubuntu to debian since I noticed that debian seems to have missing drivers for blutooth devices, and sometimes missing wifi drivers, ubuntu seems to work better with both of these things, like on my older laptos like the dell it seems to matter.

**Reply by ghost** [reply]
Hi @maximalEbooks! An IvyBridge 3930K would work flawlessly with the rammus image (unless it has NVidia graphics of course)
I think that intel i7 3930K gen 2 is under Sandy Bridge not IvyBridge but I could be wrong about that?

**Reply by tejasraman** [reply]
You're right... most 3rd gens are IvyBridge, This one is Sandy Bridge-**X**... but rammus should still work. (Works fine on my sandu bridge laptop from 2011)
